addon/components/models-table.js

```diff
@@ -53,6 +53,7 @@ const defaultMessages = {
   'columns-hideAll': 'Hide All',
   'columns-restoreDefaults': 'Restore Defaults',
   tableSummary: 'Show %@ - %@ of %@',
+  loading: 'Loading...',
```

Comment:
It should be checked how the default texts could be replaced with custom ones in order to use translations